src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -101,15 +99,18 @@ Here are some example user management procedures:
 <CollapserGroup>
   <Collapser
     id="add-users"
-    title="Add and edit users"
+    title="Add, edit, and delete users"
   >
     To add or edit users, use the [User management](#where) UI.
 
-    To add users there:
+    To add users:
 
     1. If you have multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-pricing-users/configure-authentication-domains), choose one from the authentication domain dropdown.
     2. Click **Add user**.
-    3. Complete the process, including choosing [user type](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) and [user group](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group).
+    3. Complete the process, including choosing [user type](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) and [user group](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group).    
```

Comment:
choosing _the_ user type?

Reply:
think that could go either way, think it's fine